### ISSUE TITLE
Using other CLIP model rather than ViT-H-14

### ISSUE THREAD

**Issue by Cherry-first** [concept-fusion/concept-fusion#13]
Hi,
 Did you use other CLIP model, like ViT-L-16 to check the results?
 Whether does only 'ViT-H-14' perform well?

**Comment by krrish94** [comment]
Yes, you can absolutely use other CLIP models. Alternatively, you could use models like SigLip and PerceptionEncoder too
